airflow/models/xcom_arg.py

```diff
@@ -59,9 +61,9 @@ class XComArg(DependencyMixin):
     :type key: str
     """
 
-    def __init__(self, operator: Union[BaseOperator, MappedOperator], key: str = XCOM_RETURN_KEY):
-        self._operator = operator
-        self._key = key
+    def __init__(self, operator: "Union[BaseOperator, MappedOperator]", key: str = XCOM_RETURN_KEY):
```

Comment:
```suggestion
    def __init__(self, operator: Union[BaseOperator, MappedOperator], key: str = XCOM_RETURN_KEY):
```

Any reason for this change?

Reply:
To avoid uncessary imports/reduce chance import cycles.

Reply:
After #20945 we can probably change this to `Operator`, which would not cause import cycles.